Cargo.toml

```diff
@@ -107,5 +107,5 @@ zigzag = "0.1.0"
 warnings = "deny"
 
 [workspace.lints.clippy]
-all = "deny"
+all = { level = "deny", priority = -1 }
```

Comment:
What does this do?

Reply:
This is part of us using nightly, seems like they now complain if two rules have same "priority" so you make this priority lower so it can be overriden by individual rules